gen/enum_test.go

```diff
@@ -96,6 +96,37 @@ func TestEqualsEnumWithValues(t *testing.T) {
 	}
 }
 
+func TestValueOfEnumWithHexValues(t *testing.T) {
+	tests := []struct {
+		e te.EnumWithHexValues
+		i int32
+	}{
+		{te.EnumWithHexValuesX, 291},
+		{te.EnumWithHexValuesY, 1110},
+		{te.EnumWithHexValuesZ, 1929},
```

Comment:
Could we just use hex forms here? It'll make it obvious what this is matching against in the Thrift file.
```suggestion
		{te.EnumWithHexValuesX, 0x123},
		{te.EnumWithHexValuesY, 0x456},
		{te.EnumWithHexValuesZ, 0x789},
```